### ISSUE TITLE
Upgrade to jakarta.xml.bind v2.3.3

### ISSUE THREAD

**Issue by mattrpav** [highsource/jaxb-tools#191]
Update to the latest jakarta version of jaxb binding jars

**Comment by weberhofer** [comment]
This is related to #148

**Comment by laurentschoelens** [comment]
@mattrpav : this is fixed due to 2.0.2 plugin version no ?

**Comment by laurentschoelens** [comment]
```
<jaxb23.version>2.3.7</jaxb23.version>
<jaxb-api.version>2.3.3</jaxb-api.version>
<jaxb.version>${jaxb23.version}</jaxb.version>
```
